packages/core/scripts/identifier-umip/1_Propose.js

```diff
@@ -70,6 +76,24 @@ async function runExport() {
   console.log(`
   Identifiers Proposed`);
   console.table(identifierTable);
+
+  console.log("Transaction: ", txn?.tx);
+
+  const finder = await Finder.deployed();
+  const oracleAddress = await finder.getImplementationAddress(web3.utils.utf8ToHex(interfaceName.Oracle));
+  console.log(`Governor submitting admin request to Voting @ ${oracleAddress}`);
+
+  const oracle = await Voting.at(oracleAddress);
```

Comment:
nit: why not `Voting.deployed()`?

Reply:
No particular reason, just what was copied and pasted from elsewhere.